CompetitionTemplate/src/competition/subsystems/arm/arm_commands/ArmManualControlCommand.java

```diff
@@ -26,7 +26,24 @@ public void initialize() {
 
     @Override
     public void execute() {
-        armSubsystem.setArmMotorPower(-oi.operatorJoystick.getVector().y/2);
+        double desire = -oi.operatorJoystick.getVector().y;
+        // offset by range joystick kicks in
+        if(desire > 0) {
```

Comment:
Won't this make the arm move in the opposite direction if the joystick value is within 0.15 of 0?